src/wazuh_db/wdb_fim.c

```diff
@@ -453,9 +453,16 @@ int wdb_fim_insert_entry2(wdb_t * wdb, const cJSON * data) {
             } else if (strcmp(element->string, "value_type") == 0) {
                 sqlite3_bind_text(stmt, 20, element->valuestring, -1, NULL);
             } else if (strcmp(element->string, "inode") == 0) {
-                // New 4.13.0 version of the message, inode is a string.
+                // New 4.13.1 version of the message, inode is a string.
+                char* end_ptr = NULL;
```

Comment:
If these changes are to validate my suggestion here 

https://github.com/wazuh/wazuh/pull/30718#discussion_r2205555336

the seg fault will still happen